src/common/ZulipStatusBar.js

```diff
@@ -15,6 +15,7 @@ class ZulipStatusBar extends PureComponent {
   };
 
   props: {
+    fullScreen?: boolean,
```

Comment:
Can you elaborate why we need the distinction of 'hidden' and 'fullScreen'?

Reply:
I was thinking to make it like WhatsApp Lightbox, but they are using [Android Full Screen activity](https://developer.android.com/training/system-ui/immersive.html) where status bar overlap screen content (image). Which is not possible in react-native. 

Removed it. Now it looks like FB messenger.